core/example/scafacos_example.cpp

```diff
@@ -0,0 +1,290 @@
+#include <Cabana_Types.hpp>
+#include <Cabana_AoSoA.hpp>
+#include <Cabana_MemberDataTypes.hpp>
```

Comment:
@sslattery what is the new header to use here?

Reply:
`Cabana_MemberTypes.hpp`